sdk/keyvault/Microsoft.Azure.KeyVault.Extensions/tests/Microsoft.Azure.KeyVault.Extensions.Tests.csproj

```diff
@@ -12,11 +12,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.Azure.KeyVault\Microsoft.Azure.KeyVault.csproj" />
-    <ProjectReference Include="..\Microsoft.Azure.KeyVault.Cryptography\Microsoft.Azure.KeyVault.Cryptography.csproj" />
-    <ProjectReference Include="..\Microsoft.Azure.KeyVault.Extensions\Microsoft.Azure.KeyVault.Extensions.csproj" />
-    <ProjectReference Include="..\Microsoft.Azure.KeyVault.Core\Microsoft.Azure.KeyVault.Core.csproj" />
-    <ProjectReference Include="..\Microsoft.Azure.KeyVault.TestFramework\Microsoft.Azure.KeyVault.TestFramework.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Azure.KeyVault\src\Microsoft.Azure.KeyVault.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Azure.KeyVault.Cryptography\src\Microsoft.Azure.KeyVault.Cryptography.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Azure.KeyVault.Extensions\src\Microsoft.Azure.KeyVault.Extensions.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Azure.KeyVault.Core\src\Microsoft.Azure.KeyVault.Core.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Azure.KeyVault.TestFramework\src\Microsoft.Azure.KeyVault.TestFramework.csproj" />
```

Comment:
We shouldn't have the TestFramework project under the root of keyvault. We should move it under one of the other tests folders. I would suggest putting it in a sub-folder under Microsoft.Azure.KeyVault\tests\TestFramework. Note if you do that you will need to exclude that directory from being automatically added to the Microsoft.Azure.KeyVault.Tests project.

Reply:
Done